lighthouse-core/report/html/renderer/report-renderer.js

```diff
@@ -157,6 +157,39 @@ class ReportRenderer {
     return container;
   }
 
+  /**
+   * @param {LH.ReportResult} report
+   * @param {CategoryRenderer} categoryRenderer
+   * @param {Record<string, CategoryRenderer>} specificCategoryRenderers
```

Comment:
`categoryRenderer` and `specificCategoryRenderers` probably need docstrings (sorry for that second name :)